schedule/migration/openldap_to_389ds.yaml

```diff
@@ -0,0 +1,7 @@
+name:   openldap_to_389ds 
+description:    >
+    This is for  openldap to 389ds migration tools test
```

Comment:
extra whitespace

Reply:
update

Reply:
there is still a double space between "for" and "openldap"

Reply:
Also I think a description with "This is for" is not really helpful. Better describe what this scenario is actually doing